virt/virt-2-6-release-notes.adoc

```diff
@@ -75,6 +75,7 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 == Notable technical changes
 
 //CNV-9339: When deploying from the GUI, the CR for the HCO is now created during subscription
+* The procedure for xref:../virt/install/installing-virt-web.adoc#installing-virt-web[installing {VirtProductName} in the web console] has been streamlined. You can now create the HyperConverged Cluster Operator custom resource immediately after installing the {VirtProductName} Operator by clicking *Create HyperConverged*.
```

Comment:
Should the name of the CR object be in back ticks? `HyperConverged Cluster Operator`?

Reply:
I am torn on this because the CR type is actually just "HyperConverged." Maybe I'll change it to that to avoid confusion.